content/cloud-docs/api-docs/teams.mdx

```diff
@@ -154,6 +158,7 @@ Properties without a default value are required.
     "type": "teams",
     "attributes": {
       "name": "team-creation-test",
+      "sso-team-id": "cb265c8e41bddf3f9926b2cf3d190f0e1627daa4",
```

Comment:
Just wanted to call out that I **didn't** add a _The sso-team-id attribute is only returned if this team's organization is in the Terraform Cloud Business tier._ note here b/c it is called out in the table immediately above

Reply:
okay that's fair and makes sense to me!